docs/test-cases/vcf-highlight/Conditions/Highlight vcf with simple conditions.md

```diff
@@ -0,0 +1,77 @@
+# Highlighting variations depending on the set conditions
+
+Test verifies
+ - highlighting background of variants in Variants panel and VCF track on Browser in dependence of specified simple conditions in JSON file
+
+**Prerequisites**:
+
+1. dataset = **dm6_data**, .vcf = **[agnts3.09-28.trim.dm606.realign.vcf](https://ngb-oss-builds.s3.amazonaws.com/public/data/demo/ngb_demo_data/agnts3.09-28.trim.dm606.realign.vcf.gz)**
+2. **interest_profiles.json** that contains **'Complex Conditions'** profile with conditions.
+```
+{
+  "Simple Conditions": {
+    "is_default": false,
+    "conditions": [
+      {
+        "highlight_color": "bc32d0",
+        "condition": "BaseQRankSum == -0.058"
+      },
+      {
+        "highlight_color": "ffff00",
+        "condition": "AN != 1"
+      },
+      {
+        "highlight_color": "add8e6",
+        "condition": "FS > 96.900"
+      },
+      {
+        "highlight_color": "dddddd",
+        "condition": "MQ >= 91.95"
+      },
+      {
+        "highlight_color": "78df72",
+        "condition": "BaseQRankSum < -11.900"
+      },
+      {
+        "highlight_color": "d3ca3b",
+        "condition": "QD <= 0.32"
+      },
+      {
+        "highlight_color": "f1eefb",
+        "condition": "AC  in [1,1,3]"
+      },
+      {
+        "highlight_color": "e89baa",
+        "condition": "MLEAC notin [1]"
+    }
+    ]
+  }
+}
+```
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login to NGB as system admin user | |
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **select the profile** field is displayed |
+| 4 | Choose **Complex Conditions** profile in dropdown list of **'select the profile'** field **(if it is not default)** | 
```

Comment:
Remove the phrase `**(if it is not default)**` - profile from the prerequisites is not default (according to the value of the key `is_default` )